c/CMakePresets.json

```diff
@@ -0,0 +1,70 @@
+{
+    "version": 3,
+    "cmakeMinimumRequired": {
+        "major": 3,
+        "minor": 21,
+        "patch": 0
+    },
+    "configurePresets": [
+        {
+            "name": "debug",
+            "displayName": "debug, all drivers, with tests, with ASan/UBSan (not usable from Python)",
+            "generator": "Ninja",
+            "cacheVariables": {
+                "CMAKE_BUILD_TYPE": "Debug",
+                "CMAKE_EXPORT_COMPILE_COMMANDS": "ON",
+                "ADBC_BUILD_TESTS": "ON",
+                "ADBC_DRIVER_FLIGHTSQL": "ON",
+                "ADBC_DRIVER_MANAGER": "ON",
+                "ADBC_DRIVER_POSTGRESQL": "ON",
+                "ADBC_DRIVER_SNOWFLAKE": "ON",
+                "ADBC_DRIVER_SQLITE": "ON",
+                "ADBC_USE_ASAN": "ON",
+                "ADBC_USE_UBSAN": "ON"
+            }
+        },
+        {
+            "name": "debug-python",
+            "displayName": "debug, all drivers, with tests, without ASan/UBSan (usable from Python)",
```

Comment:
I think you could in theory use ASAN/UBSan from python. NumPy has a CI job as such:

https://github.com/numpy/numpy/pull/24208/files

Reply:
Hmm, that LD_PRELOAD seems like it's the thing I've been missing before...

Reply:
Deferred to #1044